src/kernel-install/40-early-cpio.install

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# This file is part of systemd.
+#
+# systemd is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# systemd is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with systemd; If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+COMMAND="${1:?}"
+
+if [ "$COMMAND" = "add" ]; then
+    for ucode in "intel-ucode.img" "amd-ucode.img" "intel-uc.img" "amd-uc.img" "early_ucode.cpio" "microcode.cpio"; do
+        for dir in "$KERNEL_INSTALL_BOOT_ROOT" "/boot"; do
+        if [ -f "${dir}/${ucode}" ]; then
+            install "${dir}/${ucode}" "${KERNEL_INSTALL_STAGING_AREA}/early-cpio-${ucode}" || {
+                echo "Error: could not copy '$ucode' to the staging area." >&2
+                exit 1
+            }
+        fi
```

Comment:
Missing indentation and to mimic the style of other install invocations, also add the mode
```suggestion
            if [ -f "${dir}/${ucode}" ]; then
                install -m 0644 "${dir}/${ucode}" "${KERNEL_INSTALL_STAGING_AREA}/early-cpio-${ucode}" || {
                    echo "Error: could not copy '$ucode' to the staging area." >&2
                    exit 1
                }
            fi
```